### ISSUE TITLE
Update serendipity_event_social to latest shariff version

### ISSUE THREAD

**Issue by yellowled** [s9y/Serendipity#553]
`serendipity_event_social` bundles verson 1.23.2 of https://github.com/heiseonline/shariff – latest release at the time of writing is 3.0.1. Starting with 3.x, shariff bundles the fontawesome iconfont instead of getting it from a CDN, so this is relevant for DSGVO/GDPR.

Updating should be as easy as adding a new build of the CSS, JS and the new font files. For optimiziation I would suggest to:

* put the font files in a separate subdirectory
* drop the EOT (I6-8 only) and SVG (legacy iOS only) files altogether
* also remove EOT and SVG from CSS

[Browser support](https://css-tricks.com/snippets/css/using-font-face/#article-header-id-2) for proposed usage

**Comment by onli** [comment]
No, there is no that variable it the CSS hook is sued like here. But you could introduce one: Update the css to use {TEMPLATE_PATH}, replace the variable in https://github.com/s9y/additional_plugins/blob/7e8ad5365a6c2f117c091d3a2543af26d67c5634/serendipity_event_social/serendipity_event_social.php#L153 with the correct path.

Regardless, thanks for updating the plugin!

**Comment by onli** [comment]
no, we already have the variable. $serendipity['template'] or something like that. You'd just have to add it to the css, by replacing the placeholder, at the line. I'm running late, will try to show an example later.
